tests/pytests/unit/modules/test_zfs.py

```diff
@@ -18,6 +18,8 @@
 from tests.support.mock import MagicMock, patch
 from tests.support.zfs import ZFSMockData
 
+pytestmark = [pytest.mark.slow_test]
```

Comment:
```suggestion
pytestmark = [pytest.mark.slow_test,]
```
Better yet, even if it's a single item list, always add the `,`. Its more readable when formatted vertically.

Reply:
I don't agree with this, we have [item] all over salt, pylint is okay with [item] but [item,] will be formatted to multi line